File: app/assets/javascripts/backbone/models/note.js
(function($) {

  NotesApp.Models.Note = Backbone.Model.extend({
    urlRoot: '/notes',
    defaults: {
      "title"   : "",
      "content" : "",
      "pos_x"    : "0px",
      "pos_y"     : "47px",
      "z_index" : 1
    }
  });

  NotesApp.Collections.Notes = Backbone.Collection.extend({
    model: NotesApp.Models.Note,
    url: '/notes'
  });

})(jQuery);

File: app/models/note.rb
class Note < ActiveRecord::Base

  attr_accessible :content, :title, :pos_x, :pos_y, :z_index

  def as_json(options = nil)
    super({
      :only => [:id, :content, :title, :pos_x, :pos_y, :z_index]
    }.merge(options))
  end

end
